tests/sympc/tensor/replicatedshare_tensor_test.py

```diff
@@ -136,3 +138,30 @@ def test_hook_property(get_clients) -> None:
 
     assert (rst.T.shares[0] == x.T).all()
     assert (rst.T.shares[1] == y.T).all()
+
+
+@pytest.mark.parametrize("parties", [2, 3, 5, 7, 11])
```

Comment:
Hrishi , I think we agreed to include a sanity check in the future preventing initializing session of falcon with other `3` parties. This would conflict that scenario.

Reply:
Hmmmm, true. RIght now using Falcon as the protocol is just a proxy for testing RSTensor. I think the default protocol we discussed in https://github.com/OpenMined/SyMPC/issues/218 should serve the purpose when the protocol does not serve a specific number of parties.